predicthq/endpoints/v1/events/schemas.py

```diff
@@ -58,6 +58,30 @@ class Options:
     type = StringType()
     formatted_address = StringType()
 
+class Geometry(Model):
+
+    class Options:
+        serialize_when_none = False
+
+    type = StringType()
+    coordinates = ListType(FloatType())
```

Comment:
Does the case of a multipolygon needs to be handled differently? Is it going to be a list of list of float?

Reply:
```
"geo": {
                "geometry": {
                    "type": "Point",
                    "coordinates": [
                        -71.084177,
                        42.347489
                    ]
                }
            }
```
current geo looks like this. I would suppose we would use a different field other than coordinates here for polygon because a list type doesn't work for that and changing type for this would be a breaking change

Reply:
need @ron-phq 's input here

Reply:
updated

Reply:
Looks good Chloe.

How come you decided to implement MultiPoint but not LineString or some of the other types? FWIW we curently only expect to have Point, Polygon, and MultiPolygon in the near future.

Reply:
sorry the naming is really bad 😓 , both MultiString and LineString are validated under the MultiPoints model, do you have better suggestion for the naming?

Reply:
same for ['MultiLineString', 'Polygon'] 😓 , now i see the naming is horrible

Reply:
Ahh, sorry I missed that. Just some ideas:

- Is there a way to create model aliases? E.g. something like creating an alias class/model called = MultiLineString which is the same as the Polygon model.
- Or try inheritence? E.g. Polygon and MultiLineString inherit from the same model (called something like CollectionMultiPoints or ListMultiPoints)
- Or create aliases for the types you're using for `coordinates`, e.g. alias `ListType(FloatType())` to Coordinates, then you can use `ListType(Coordinates)` etc.
- Try borrow Ivan's naming from the event geo utils 😅